### ISSUE TITLE
Breaks url when request contains [ ]

### ISSUE THREAD

**Issue by chowaikong** [ihsanbal/LoggingInterceptor#22]
I'm defining:
```java
@GET("www.api.com/get/")
Call getRequest(@Query("key[one]")  String one)
```
When printing URL in the logcat. it goes like this `http://www.api.com/get/key[one]=one`, the clickable part is `http://www.api.com/get/key`, and it stopped, so it's not a fully clickable url in my case, can it be fixed?  Thanks.

**Comment by ihsanbal** [comment]
Not relevant to the library, It's relevant to the logcat.

**Comment by ihsanbal** [comment]
fixed with #27
